finalproject/designreview1.md

```diff
@@ -1,23 +1,56 @@
 # Design Review 1
 
-###This will take place in class on Friday, April 1
-
-Details will be posted here soon
-
-<!-- You will have **8 minutes** to present the following information:
-
-- Elevator pitch - A brief (30-45 second) pitch of your project idea.
-What is it? Why is it exciting?
-
-- In-depth project description including:
-  - system and infrastructure architecture diagrams.
-  - UI Mockups (sketches/wireframes)
-
-- Project roadmap with big picture features and a rough timeline -
-In what order will you tackle things? How long do you expect each aspect to take?
-
-- What are you the most worried about - The big unknowns, assumptions, showstoppers
-in your project. What have you done so far to understand these better, what are your next steps?
-
-There will be a few minutes at the end for questions and to collect feedback from the class and teaching team.
- -->
\ No newline at end of file
+### Format
+We’ll give you 10 min at the beginning of class to prepare, then the reviews will start at 3:30 in AC328 and three other classrooms in the AC. You’ll present to 1-2 other teams and 1-2 teaching team members, and you’ll have 15 minutes to do whatever will be most helpful for you moving forward -- get feedback or advice, do a codesign, do a code review... up to you. The flexibility we’re giving you means that we trust you to use the time productively.
```

Comment:
Should they not spend any time before class preparing? This sentence could be taken to imply as such. Is there a set amount of presentation time before the 15 minute "activity" time, are those both in the same 15 minutes?